addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -388,7 +388,10 @@ export class HtmlField extends Component {
         $odooEditor.removeClass('odoo-editor-editable');
         $editable.html(html);
 
+        // Prevent history reverts.
+        this.wysiwyg.odooEditor.observerUnactive('toInline');
         await toInline($editable, this.cssRules, this.wysiwyg.$iframe);
+        this.wysiwyg.odooEditor.observerActive('toInline');
```

Comment:
Are there other places where we call `toInline` but without stopping obversations ? Would this be something we might want ? I'd say no because, if I understood correctly, the `toInline` function _cannot_ work while the observer is active so why not put those calls into the `toInline` function directly ?

Reply:
It can't indeed. This should be in `toInline` itself.

Reply:
Although technically you might want to call `toInline` on an element that is not editable, in which case you wouldn't need to do that. But that use case is untested and definitely not guaranteed to work.

Reply:
True. Let's keep it outside, then. Handling observing is not the responsibility of this function.